src/app/admin/pages/learning-objects/learning-objects.component.ts

```diff
@@ -22,10 +24,18 @@ export class LearningObjectsComponent {
   activeLearningObject;
   adminStatusList =  Object.keys(LearningObject.Status);
   selectedStatus: string;
+  sub: Subscription;
+
+  ngOnInit(): void {
+    this.sub = this.route
```

Comment:
This should employ a `takeUntil` pattern wherein the component has a subject that emits when it's destroyed and all subscriptions in the component listen for that event and unsubscribe.